components/List/List.js

```diff
@@ -0,0 +1,28 @@
+import data from "../../lib/dummyArray";
+import {
+  StyledTable,
+  StyledTableCell,
+  StyledTableHeading,
+  StyledTableRow,
+} from "./styles";
+
+export default function List() {
+  return (
+    <StyledTable>
+      <tbody>
```

Comment:
There is a typo. <tbody> I guess that is why Vercel is failing to render the code.

Reply:
Hey Volker,
first of all thanks for the review.
<tbody> is an html element used for tables, witthout it my table doesnt look good anymore.. 
Idk why the Vercel is failing to render the code. When i start the server locally everything works fine.